### PR TITLE
Vlad dev

### PR DESCRIPTION
**Pull request by HornetPilot01**

### CODE REVIEW

File: Bon_Voyage/ClientApp/src/views/othersViews/HomePage/HomePage.js [hunk 1]
```diff
@@ -39,9 +41,65 @@ class HomePage extends Component {
...
+                    <div class="site-mobile-menu-header">
+                      <div class="site-mobile-menu-close mt-3">
+                        <span class="icon-close2 js-menu-toggle"></span>
+                      </div>
+                    </div>
+                    <div class="site-mobile-menu-body"></div>
+                  </div>
+
+
+
```
Comment (Karonuk) [inline Bon_Voyage/ClientApp/src/views/othersViews/HomePage/HomePage.js:59]: delete this

File: Bon_Voyage/ClientApp/src/views/othersViews/HomePage/HomePage.js [hunk 2]
```diff
@@ -39,9 +41,65 @@ class HomePage extends Component {
...
+                    </div>
+                  </div>
+                </div>  */}
+
+
+
+
+
+
+
```
Comment (Karonuk) [inline Bon_Voyage/ClientApp/src/views/othersViews/HomePage/HomePage.js:101]: delete this

File: Bon_Voyage/ClientApp/src/views/othersViews/LoginPage/reducer.js [hunk 3]
```diff
@@ -12,6 +12,8 @@ export const LOGIN_POST_SUCCESS = "LOGIN_POST_SUCCESS";
 export const LOGIN_POST_FAILED = "LOGIN_POST_FAILED";
 export const LOGIN_SET_CURRENT_USER = "login/SET_CURRENT_USER";
 
+
+
```
Comment (Karonuk) [inline Bon_Voyage/ClientApp/src/views/othersViews/LoginPage/reducer.js:16]: delete this

File: Bon_Voyage/MediatR/Home/ViewModel/HomeInfoViewModel.cs [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+using System.Threading.Tasks;
+
+namespace Bon_Voyage.MediatR.Home.ViewModel
+{
+    public class HomeInfoViewModel
+    {
+        public List<Bon_Voyage.DB.Entities.Country> Countries { get; set; }
+        //public string Top3Photos { get; set; }
+        public List<Ticket> TopTickets { get; set; }
+        public List<Ticket> TopHotTickets { get; set; }
```
Comment (Karonuk) [inline Bon_Voyage/MediatR/Home/ViewModel/HomeInfoViewModel.cs:14]: there must be a view models but not entities

File: Bon_Voyage/MediatR/Hotel/ViewModels/HotelViewModel.cs [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+namespace Bon_Voyage.MediatR.Hotel.ViewModels
+{
+    public class HotelViewModel
+    {
+        public string Id { get; set; }
+        public string Name { get; set; }
+        public int Stars { get; set; }
+        public string Description { get; set; }
+        public string CityId { get; set; }
```
Comment (Karonuk) [inline Bon_Voyage/MediatR/Hotel/ViewModels/HotelViewModel.cs:14]: why are u sending city id to frontend?

File: Bon_Voyage/MediatR/Cities/ViewModels/CityViewModel.cs [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Bon_Voyage.MediatR.Cities.ViewModels
+{
+    public class CityViewModel
+    {
+        public string Id { get; set; }
+        public string Name { get; set; }
+        public string CountryId { get; set; }
```
Comment (Karonuk) [inline Bon_Voyage/MediatR/Cities/ViewModels/CityViewModel.cs:12]: why are u sending country id to frontend?

File: Bon_Voyage/MediatR/Cities/Commands/AddCity/AddCityCommand.cs [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+                    Name = request.Name,
+                    CountryId = request.CountryId
+                });
+                _context.SaveChanges();
+
+                return true;
+            }
+        }
+    }
+}
```
Comment (Karonuk) [inline Bon_Voyage/MediatR/Cities/Commands/AddCity/AddCityCommand.cs:48]: it's not your task part

**Review by Maksym-Rohach — APPROVED** [review]

**Pull request merged**